messages/cardano.proto

```diff
@@ -30,6 +30,13 @@ enum CardanoNetwork {
   CardanoTestnet = 1;
 }
 
+enum CardanoDRepType {
+  KeyHash = 0;
+  ScriptHash = 1;
+  AlwaysAbstain = 2;
+  AlwaysNoConfidence = 3;
```

Comment:
Unfortunately protobuf enum fields live in the global namespace, so all fields need a namespace prefix :( Please add `CARDANO_DREP_TYPE_`. In Rust, the protobuf generator will strip the prefix again I think, so it's not too bad.

If this message will never be used outside of the VoteDelegation message, you can also define it within VoteDelegation. In that case, you don't need the prefix, as the enum will be scoped.

Afaik the naming convention for enum fields is all uppercase too, so `KEY_HASH`, etc.

Reply:
I do not think that in forseeable future with the current business case I was told of this will be used elsewhere, so I moved it into `VoteDelegation`. I put it on top with the casing based on other protobuf files and the naming in `CardanoNetwork` to keep consistency, but this should be nicer.

Reply:
Thanks. Indeed, CardanoNetwork uses a different style unfortunately. I checked, most other enums in the project are uppercase.